docs/contributing-test-results.md

```diff
@@ -13,7 +13,7 @@ The process is as follows:
 - Run e2e tests (we recommend using [kubetest](/kubetest/README.md))
 - Store test results in accordance with [Gubernator's GCS Layout](/gubernator/README.md#gcs-layout)
   - Unfortunately this part is currently left as an exercise to the reader. We use
-    [bootstrap](/bootstrap/README.md) to facilitate this and are in the midst of
+    [bootstrap](/jenkins/README.md) to facilitate this and are in the midst of
     rewriting it to better support external usage.
```

Comment:
Let's link to jenkins/bootstrap.py instead. The Jenkins readme isn't very helpful for this.

Reply:
@BenTheElder  yes,,may be you are right, i change it, ths